edx_rbac/decorators.py

```diff
@@ -0,0 +1,35 @@
+"""
+Taken from https://github.com/escodebar/django-rest-framework-rules/blob/master/rest_framework_rules/decorators.py.
+"""
+from __future__ import absolute_import
+
+
+def permission_required(*permissions, **decorator_kwargs):
```

Comment:
In general, if you could write a test or 2 using the concrete test classes in this package's test module, that would be 👍 . I feel like that might require a views.py file in `tests/`, which might be a heavier lift, but I think it would be worth it to know that things are functioning as expected in a test case here